fr/administrator-guide/configuring-the-catalog/csw-configuration.rst

```diff
@@ -1,4 +1,22 @@
 .. _csw-configuration:
 
 Configurer le CSW
-#################
\ No newline at end of file
+#################
+
+Afin d'offrir un servie CSW au travers du catalogue, ce dernier doit être 
+activé. Pa défaut, le CSW est non-actif. L'activation se fait de manière très 
```

Comment:
CSW is enables by default https://github.com/geonetwork/core-geonetwork/blob/develop/web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql#L583